docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
-# Confidence Intervals
-By navigating to the detail view of any experiment, you can view the effect that any of the treatments have had on each of the selected metrics for that experiment. You can get to the detail view of any experiment by clicking the **Experiments** tab on the left panel and then clicking on the experiment you're interested in from the list. The effect on each metric comes with an accompanying confidence interval. This confidence interval tells you the upper and lower bound of the effect on the metric in addition to the point estimate of the effect. We show confidence intervals for the ratio between treatment and control, rather than their absolute difference.
+# Lift Estimates and Confidence Intervals
+By navigating to the detail view of any experiment, you can view the effect that any of the treatments have had on each of the selected metrics for that experiment. You can get to the detail view of any experiment by clicking the **Experiments** tab on the left panel and then clicking on the name of the experiment you're interested in from the list. In order to provide effect estimates consistently across all metric types (counts, rates, percentages, etc.), we use the **relative lift** between treatment and control (that is, $\frac{Treatment - Control}{Control}$), rather than their absolute difference.
 
-If the change is positive and a positive change has been marked as good, the interval shows up in green (red if positive is bad). The opposite is true if the change is negative. The interval will be gray if the confidence range includes 0, possibly indicating that the effect of the variation on that metric was not significant.
+The effect on each metric comes with an accompanying **confidence interval**, showing the range of values we consider plausible at the specified **significance level**, which defaults to 5%. The significance level represents how likely a treatment that was identical to the control (that is, an A/A test) would show an effect at least as large as what you've observed in your experiment.[^1] A *higher* significance level means that you are willing to accept more risk of seeing an effect that isn't actually there[^2], but also that you can make a decision with less data; it will also mean that the confidence intervals will be narrower.
+
+If zero falls within the confidence interval, then we cannot rule out (at the given significance level) the possibility that the treatment is no different from control. In that case, the confidence interval will be colored gray. On the other hand, if the entire confidence interval is above or below zero, then the effect is *statistically significant*. In that case, the confidence interval and the lift estimate box will be colored <span class="positive-change-green-bg">green</span> if the change is good, and <span class="negative-change-red-bg">red</span> if the change is bad. If the change is good, there will also be a 🎉 symbol in the lift estimate box.
```

Comment:
Here too I think we can try to avoid statistically significant -- we can talk about the connection with hypothesis tests in the more detailed section below.

Reply:
How should we refer to a metric that has green/red highlighting? We need some way to refer to that, so that we can explain what is happening. I'm happy to use another term than "statistically significant", but it seems like it's easier to at least use a term people will have heard of before than to create a new one

Reply:
Something like this maybe:

> The confidence interval method generates a confidence interval that contains the true effect with probability at least 95% (or sequence of...), thereby suggesting values outside the confidence interval are implausible. As a special case, we color the confidence interval green or red (depending on desired change) if the confidence interval does not contain zero; or that there is no difference between treatment and control.

Potentially we could add something like:
> Those familiar with hypothesis testing will recognize the correspondence between the confidence interval not containing zero and rejecting the null hypothesis that there is no difference in effect between treatment and control.